definitions/marts/looker_studio_marts/ecf/ls_ecf_cohort_corruption_checker.sqlx

```diff
@@ -159,90 +157,60 @@ WITH
-            AND dec_date_short < started_dec_start_date)
-          OR (declaration_type = 'retained-1'
-            AND dec_date_short < ret1_dec_start_date)
-          OR (declaration_type = 'retained-2'
-            AND dec_date_short < ret2_dec_start_date)
-          OR (declaration_type = 'retained-3'
-            AND dec_date_short < ret3_dec_start_date)
-          OR (declaration_type = 'retained-4'
-            AND dec_date_short < ret4_dec_start_date)
-          OR (declaration_type = 'completed'
-            AND dec_date_short < completed_dec_start_date))))) THEN "in both checks"
-      WHEN suggested_cohort_from_declaration <> cohort
-    AND ((schedule_identifier LIKE 'ecf-standard%'
-        AND ((declaration_type = 'started'
-            AND (dec_date_short NOT BETWEEN started_dec_start_date
-              AND started_dec_milestone_date))
-          OR (declaration_type = 'retained-1'
-            AND (dec_date_short NOT BETWEEN ret1_dec_start_date
-              AND ret1_dec_milestone_date))
-          OR (declaration_type = 'retained-2'
-            AND (dec_date_short NOT BETWEEN ret2_dec_start_date
-              AND ret2_dec_milestone_date))
-          OR (declaration_type = 'retained-3'
-            AND (dec_date_short NOT BETWEEN ret3_dec_start_date
-              AND ret3_dec_milestone_date))
-          OR (declaration_type = 'retained-4'
-            AND (dec_date_short NOT BETWEEN ret4_dec_start_date
-              AND ret4_dec_milestone_date))
-          OR (declaration_type = 'completed'
-            AND (dec_date_short NOT BETWEEN completed_dec_start_date
-              AND completed_dec_milestone_date))))
-      OR (schedule_identifier LIKE ANY ('ecf-extended%',
-          'ecf-reduced%',
-          'ecf-replacement%')
-        AND ((declaration_type = 'started'
-            AND dec_date_short < started_dec_start_date)
-          OR (declaration_type = 'retained-1'
-            AND dec_date_short < ret1_dec_start_date)
-          OR (declaration_type = 'retained-2'
-            AND dec_date_short < ret2_dec_start_date)
-          OR (declaration_type = 'retained-3'
-            AND dec_date_short < ret3_dec_start_date)
-          OR (declaration_type = 'retained-4'
-            AND dec_date_short < ret4_dec_start_date)
-          OR (declaration_type = 'completed'
-            AND dec_date_short < completed_dec_start_date)))) THEN "declaration date check"
-      WHEN statement_cohort <> cohort THEN "statement check"
-    ELSE
-    NULL
-  END
-    AS check_source
-  FROM
-    statement_cohort_checks
-  WHERE
-    statement_cohort <> cohort
-    OR (suggested_cohort_from_declaration <> cohort
+      fdmt.declaration_id=si.participant_declaration_id
+    WHERE /*This excludes declarations we already know do not align with the statement cohort but the financial statements have been closed so there is no action to be taken.*/
+      declaration_id NOT IN ('59e5d04c-0741-4e1d-b1c3-8467cafc3ead',

```

Comment:
Is this list likely to expand? Do we think a static file is more sustainable/logical? Will we need this list elsewhere?

Reply:
A static list is probably a much better solution, I'll go about implementing this.